lisa/src/main/java/it/unive/lisa/cfg/statement/Literal.java

```diff
@@ -19,16 +21,17 @@
 	/**
 	 * Builds the literal, consisting of a constant value. The location where this
 	 * literal happens is unknown (i.e. no source file/line/column is available).
+	 * The type of this literal is unknown (i.e. its type is {@code Untyped.INSTANCE}).
 	 * 
 	 * @param cfg   the cfg that this expression belongs to
 	 * @param value the value of this literal
 	 */
 	public Literal(CFG cfg, Object value) {
```

Comment:
There should also be a constructor with cfg, value and type (as it happens with Variable)